bedrock/firefox/templates/firefox/features/fast.html

```diff
@@ -18,6 +18,12 @@
 
 {% block body_id %}firefox-features-fast{% endblock %}
 
+{% block optimizely %}
+  {% if switch('firefox-features-optimizely', ['en-US']) %}
```

Comment:
Does this appear on the main /features/ page as well? If it's on *all* feature pages with the same switch it could go in features/base.html